src/node_options.h

```diff
@@ -195,6 +195,7 @@ class EnvironmentOptions : public Options {
   bool test_only = false;
   bool test_udp_no_try_send = false;
   std::string test_isolation = "process";
+  std::string test_runner_cwd;
```

Comment:
```suggestion
  std::string test_runner_cwd = ...;
```

Is there a way to get the CWD here?

Reply:
I tried, but from what I've seen, the `cwd` is usually retrieved via the `Environment` class.
I need to investigate this further to understand if there is a straightforward way to retrieve it.

If you have any suggestions they're more than welcome 😁

Reply:
At the moment, many parts of the test runner logic use `process.cwd` to define the paths to test resources—for example, when searching for test files under the current working directory or when printing the test coverage

considering @cjihrig feedback:
> I don't think we should add this. The `cwd` option should be supported by the `run()` API only IMO.

I'll remove the option and use a different approach

Reply:
removed the option